website/server/controllers/api-v3/user.js

```diff
@@ -223,6 +223,10 @@ api.deleteUser = {
 
     await user.remove();
 
+    if (req.body.feedback) {
```

Comment:
This new parameter needs to be documented in the comment block and some validation would not hurt too (limit length, ..)